aspnetcore/fundamentals/index.md

```diff
@@ -69,9 +69,11 @@ For more information, see <xref:fundamentals/startup>.
 
 The content root is the base path to any content used by the app, such as [Razor Pages](xref:razor-pages/index), MVC views, and static assets. By default, the content root is the same location as the app base path for the executable hosting the app.
 
-## Web root
+## Web root (webroot)
 
-The web root of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files.
+The webroot of an app is the directory in the project containing public, static resources, such as CSS, JavaScript, and image files. By default, *wwwroot* is the webroot.
+
+For Razor (*.cshtml*) files, the tilde-slash  `~/` points to the webroot. `~/` is referred to as the virtual path of the app.
```

Comment:
Leave out the second sentence in this paragraph. Nobody says "virtual path of the app".

If you want to mention virtual path (I'm not sure it's necessary), you could say "Paths beginning with `~/` are referred to as virtual paths."